samples/RoutingSample.Web/RoutingSample.Web.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
```

Comment:
This sample is referenced by the FunctionalTests, which complain if it's not strongnamed when I replace common.props with dependencies.props. I vote that we leave it as is to get this PR through and unblock the PR's below me, and we can file a bug to not reference common.props